src/test/java/org/kainos/ea/resources/JobRoleControllerTest.java

```diff
@@ -52,4 +55,40 @@ void viewRoles_ShouldReturnResponse500_WhenServiceThrowsFailedToGetJobRoles()
         Assertions.assertEquals(500, response.getStatus());
     }
 
+    @Test
+    void editRole_ShouldReturnResponse200_WhenJobRoleHasBeenEdited()
+            throws JobRoleDoesNotExistException, FailedToGetJobRole{
+        int id = 5;
+        JobRoleEditRequest jobRole = new JobRoleEditRequest("string", "string", "string", "string", 1, 1);
+
+        Mockito.when(jobRoleServiceMock.editJobRole(id, jobRole)).thenReturn(5);
+
+        Response response = jobRoleController.editJobRole(id, jobRole);
+
+        Assertions.assertEquals(200, response.getStatus());
+    }
+
```

Comment:
Missing test for 400 being returned

Reply:
Relabelled one of my 500 tests wrong so changed that to 400